addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15838,27 +15838,41 @@ msgstr ""
 
 #24171-24179 reserved for future use of lifecycle state name (to have continuous chain of GUI)
 
+#. The minimum version a dependency must meet. This is set in the <requires> tag of the dependee
+#. {3:s} reserved for "(optional)" string resource. see #24184
 #: xbmc/addons/GUIDialogAddonInfo.cpp
 msgctxt "#24180"
-msgid "Minimum: {0:s}"
+msgid "Minimum: {0:s} {3:s}"
 msgstr ""
 
+#. The minimum version, plus the version going to be installed
+#. {3:s} reserved for "(optional)" string resource. see #24184
```

Comment:
`reserved for an empty string or the " (optional)" optional string`...

Probably should be the same in each description